iree/test/e2e/xla/reverse.mlir

```diff
@@ -1,14 +1,16 @@
-// RUN: iree-run-mlir -iree-hal-target-backends=vmla %s | IreeFileCheck %s
```

Comment:
I sometimes use `iree-run-mlir` outside of lit tests for runtime work, so the changes to use `check` in each of these tests now may affect some of my workflows (https://xkcd.com/1172/). I think a better setup for my use would be to just add more models (with no lit test comments or check IR) to https://github.com/google/iree/tree/master/iree/test/models.

I'm mainly thinking of cases where I just want to run some simple .mlir with inputs and observe the outputs ("run inference") from the command line.

No action needed here, just an observation.

Reply:
Yeah makes sense. We can leave the lit lines in as well. As long as we don't care about something running on device, we can use lit tests to ensure that it still runs (which is what I've done in iree/tools/test) so we don't get bit rot.